requirements-dev.txt

```diff
@@ -7,3 +7,4 @@ pytest-cov == 2.5.1
 pytest-mock == 1.6.3
 pytest-timeout == 1.2.1
 asynctest == 0.11.1
+zipp == 3.4.1
```

Comment:
Do we use it here or was it omitted in a different PR?

Reply:
Seems to be a `pytest` dependency which gets bungled by Travis' `pip`.  I cannot reproduce the problem locally, but this made Travis happy.